frontend/src/components/Receipt/utils/receipt-calculation.ts

```
@@ -1,371 +1,662 @@
+    }
 
-  // Add get total tip and gratuity
-  const totalTip = (receipt_data.tip ?? 0) + (receipt_data.gratuity ?? 0);
+    /**
+     * Calculates fair totals with proper rounding and penny distribution.
+     */
+    export function getPersonFairTotals(
+      receiptTotal: number,
+      personTotals: Map<string, FinalLineItemTotal>
+    ): Map<string, FinalFairLineItemTotal> {
+      debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)